internal/gengapic/gengapic.go

```diff
@@ -155,9 +155,9 @@ func (g *generator) init(files []*descriptor.FileDescriptorProto) {
 			// since the field tag of Method is 2.
 			p := loc.Path
 			switch {
-			case len(p) == 2 && p[0] == 6:
+			case len(p) == 2 && p[0] == 6 && loc.LeadingComments != nil:
```

Comment:
We'd have to do this at ever branch right? I think it's probably easier to 

```go
if loc.LeadingComments == nil {
  continue
}
```

before the `switch`.

Reply:
Good call, I'll change it up.